operators/index.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 include::modules/operators-overview.adoc[leveloffset=+1]
 
-With Operators, you can create applications to monitor the running services in the cluster. Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
+Operators are designed specifically for Kubernetes-native applications to implement and automate common Day 1 operations on a {product-title} cluster, such as installation and configuration. Operators can also automate Day 2 operations, such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
```

Comment:
So is this paragraph only about cluster Operators? Because it sounds like that, but isn't specified. 

IDK the addition of this content overall is confusing, I think because of the order of it in the section and the increased length of text without headings or other formatting. Maybe if this and the next paragraph are related to the two bullets preceding, they can each be smaller sections all together.